xtend-website/_posts/releasenotes/2021-12-31-version-2-26-0.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  Xtend 2.26.0 Release Notes
+date:   2021-12-31
```

Comment:
this is just a preliminary placeholder date.
unfortunately jekyll does not accect something like XXXX